From 38cd291b6da41439aa95450a9547f7acbf633867 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@altera.com>
Date: Mon, 4 Mar 2013 11:41:01 -0600
Subject: [PATCH 056/248] FogBugz #104179: Fix SCU virtual address mappping

Fix:

BUG: mapping for 0xfffec000 at 0xfffec000 out of vmalloc space

[Original patch taken from
git://git.rocketboards.org/linux-socfpga.git socfpga-3.14]

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Dinh Nguyen <dinguyen@altera.com>
---
 arch/arm/mach-socfpga/core.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-socfpga/core.h b/arch/arm/mach-socfpga/core.h
index 96098c1..ad766d3 100644
--- a/arch/arm/mach-socfpga/core.h
+++ b/arch/arm/mach-socfpga/core.h
@@ -66,6 +66,6 @@ extern char secondary_trampoline, secondary_trampoline_end;
 extern struct dw_mci_board sdmmc_platform_data;
 extern unsigned long cpu1start_addr;
 
-#define SOCFPGA_SCU_VIRT_BASE   0xfffec000
+#define SOCFPGA_SCU_VIRT_BASE   0xfee00000
 
 #endif /* __CORE_H */
-- 
1.9.1

